source/endpoints/devices/Activator.py

```diff
@@ -40,4 +40,5 @@ def post(self, device_id, activator_id):
         action = device.get_activator(activator_id)
         value = namespace.apis[0].payload["state"]
         action.set_state_with_string(value)
-        action.perform()
\ No newline at end of file
+        #print(device.required_info)
+        action.perform(device.required_info)
```

Comment:
The activator needs the required info of the device to communicate to the peripheral.